mkdocs.yml

```diff
@@ -278,8 +278,6 @@ plugins:
       strict: false
   - redirects:
       redirect_maps:
-        'eventing/samples/sinkbinding/': 'eventing/sources/sinkbinding/README.md'
```

Comment:
Does this line need to be deleted as well? It is redirecting `eventing/samples/` not `eventing/sources/`?

Reply:
oops you are totally right, good catch